callbacks_test.go

```diff
@@ -14,30 +15,64 @@ import (
 	"github.com/Nitro/sidecar-executor/container"
 	"github.com/Nitro/sidecar/service"
 	"github.com/fsouza/go-dockerclient"
-	mesos "github.com/mesos/mesos-go/api/v0/mesosproto"
+	mesos "github.com/mesos/mesos-go/api/v1/lib"
+	"github.com/pborman/uuid"
 	director "github.com/relistan/go-director"
+	log "github.com/sirupsen/logrus"
 	. "github.com/smartystreets/goconvey/convey"
 )
 
-type dummyMesosDriver struct {
-	isStopped      bool
+func Test_shouldCheckSidecar(t *testing.T) {
```

Comment:
This whole test file took a lot of work to get running on the new driver. However, once the setup was in place, all the tests passed with only pointer/non-pointer changes.